src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -319,8 +319,9 @@ def map_keyvalue_to_featureflagvalue(keyvalue):
                     "key:{0} value:{1}\nFull exception: \n{2}".format(keyvalue.key, keyvalue.value, str(exception))
         raise ValueError(error_msg)
 
-    except:
-        logger.error("Exception while parsing feature flag. key:%s value:%s.", keyvalue.key, keyvalue.value)
-        raise
+    except Exception as exception:
```

Comment:
Its generally bad practice to catch and rethrow base exception by wrapping it into another exception. We should just log the exception and rethrow without wrapping it into ValueError

Reply:
This has been updated to only catch specific exceptions, thanks!